contrib/terminator/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "terminator"
+pkgver = "2.1.3"
+pkgrel = 0
+build_style = "python_pep517"
+make_check_target = "tests"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-flit_core",
+    "python-wheel",
+    "gettext-tiny",
+]
+checkdepends = ["python-pytest", "python-cairo", "python-gobject"]
+depends = [
+    "python-psutil",
+    "python-gobject",
+    "desktop-file-utils",
+    "gsettings-desktop-schemas",
+    "libkeybinder3",
+    "libnotify",
+    "python-dbus",
+    "python-configobj",
+    "vte-gtk3",
+    "pango",
+]
+pkgdesc = "Tiling terminal emulator application"
+maintainer = "yopito <pierre.bourgin@free.fr>"
+license = "GPL-2.0-only"
+url = "https://gnome-terminator.org"
+source = f"https://github.com/gnome-terminator/terminator/releases/download/v{pkgver}/terminator-{pkgver}.tar.gz"
+sha256 = "0ae9943f3d6b72230c14bcb355de84dd81274c033e76ca4698e80d7c93cd6ae5"
+hardening = ["vis"]
+# crossbuild: wait for crossbuilable deps
+# checks needs gtk/gobject-instrospection and xvfb
```

Comment:
cannot be done with `xvfb-run`?

Reply:
I've tried, see additional checkdepends. But the testsuite crash, not all tests are at least runned. So I keep check disabled.